docs/components/connectors/out-of-the-box-connectors/operate.md

```diff
@@ -14,13 +14,13 @@ This connector is compatible with both Camunda 8 SaaS and Camunda 8 Self-Managed
 
 :::note
 Password authentication with Operate is currently not supported.
```

Comment:
Are there plans to ever do so? Perhaps this can even be removed completely

Reply:
This note goes all the way back to version 8.4 -- not sure if a user asked and we added a note to disclose, but I can definitely remove it for now 👍